src/sql/workbench/contrib/editData/browser/editData.contribution.ts

```diff
@@ -18,6 +24,20 @@ const editDataEditorDescriptor = new EditorDescriptor(
 	'EditData'
 );
 
+const configurationRegistry = <IConfigurationRegistry>Registry.as(ConfigExtensions.Configuration);
```

Comment:
why are we adding this setting?  i.e. has this been requested by a customer?

Reply:
Yes, a customer requested this feature should be added